sorc/build_ufs.sh

```diff
@@ -32,10 +32,28 @@ COMPILE_NR=0
 CLEAN_BEFORE=YES
 CLEAN_AFTER=NO
 
-./tests/compile.sh "${MACHINE_ID}" "${MAKE_OPT}" "${COMPILE_NR}" "intel" "${CLEAN_BEFORE}" "${CLEAN_AFTER}"
-mv "./tests/fv3_${COMPILE_NR}.exe" ./tests/ufs_model.x
-mv "./tests/modules.fv3_${COMPILE_NR}.lua" ./tests/modules.ufs_model.lua
-cp "./modulefiles/ufs_common.lua" ./tests/ufs_common.lua
-cp "./modulefiles/ufs_common_spack.lua" ./tests/ufs_common_spack.lua
```

Comment:
We might need this spack module soon. I wouldn't remove it.

Reply:
@WalterKolczynski-NOAA I believe the UFS WM is carrying these around now. This was a hack in the original PR until @aerorahul pointed me to the UFS WM feature.

Reply:
This whole bit is happening inside the UFS. But just leave it, we can add the copy back when we need it.